src/main/webapp/resources/js/pages/analysis/Analysis.jsx

```diff
@@ -0,0 +1,63 @@
+/*
+ * This file is responsible for displaying the
+ * tabs required depending on the analysis state
+ * and analysis type.
+ */
+
+import { Layout, PageHeader, Skeleton } from "antd";
+import React, { Suspense, useContext } from "react";
+import {
+  IconCheck,
+  IconLoading,
+  IconWarning,
+} from "../../components/icons/Icons";
+import { ContentLoading } from "../../components/loader";
+import { AnalysisContext } from "../../contexts/AnalysisContext";
+import { blue6, green6, grey1, red6 } from "../../styles/colors";
+import AnalysisCompletePage from "./AnalysisCompletePage";
```

Comment:
Should this all be lazy loaded? Just asking since the other Analysis*Page are lazy loaded.

Reply:
Updated in 3eeda90f7688ec52a8ef8047abe0a6ddfa660f66